frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1122,13 +1122,13 @@ function ContextMenuInner(propsWithInputRef: Props) {
         const [segmentSize] = await getSegmentVolumes(
           tracingStoreUrl,
           tracingId,
-          mag.getLowestResolution(),
+          mag.getFinestResolution(),
```

Comment:
```suggestion
          magInfo.getFinestResolution(),
```

I’ve also seen resolutionInfo for this thing elsewhere. Also fine by me.

I think it would be important to distinguish that this object contains information about all the available mags (resolutions)